cleanslate/cleanslate/urls.py

```diff
@@ -19,12 +19,13 @@
 from django.views.generic import RedirectView
 from django.conf import settings
 from django.conf.urls.static import static
+from django.contrib.auth import views as auth_views
 from doma.views import home, profile, calendar, reminders, finance, edit_chore_deadline, create_chore, delete_chore, EditUserProfileView
 
 urlpatterns = [
-    url(r'^$', home, name='doma/login/'),
```

Comment:
Removing this breaks the 'home' button (the doma button in the center of the navbar). It now asks for a login on every press and upon logging in just redirects to the profile page.